requirements-workers.txt

```diff
@@ -0,0 +1 @@
+celery
```

Comment:
Don't you need celery also in the main `requirements.txt` ? I thought that was needed to actually create celery tasks.

Reply:
You're right. I'll fix it.

Reply:
It's actually not needed yet, so I'll just remove it from this PR and add it back later